tools/login/actions/login.php

```diff
@@ -32,15 +39,20 @@
 // Lecture des parametres de l'action
 
 // NOTE: à mettre dans la classe ?
+
+// urls after successfull login and successfull logout
+
+$loggedinurl = $this->GetParameter('loggedinurl');
+$loggedouturl = $this->GetParameter('loggedouturl');
+
 // url d'inscription
 $signupurl = $this->GetParameter('signupurl');
 // si pas d'url d'inscription renseignée, on utilise ParametresUtilisateur
-if (empty($signupurl) || $signupurl === "0") {
+
+if (empty($signupurl) && (!(isset ($signupurl) && $signupurl === "0"))) {
     $signupurl = $this->href("", "ParametresUtilisateur", "");
-} else {
-    if ($this->IsWikiName($signupurl, WN_CAMEL_CASE_EVOLVED)) {
+} else if (($signupurl !== "0") && ($this->IsWikiName($signupurl, WN_CAMEL_CASE_EVOLVED))) {	
```

Comment:
```suggestion
} else if ($this->IsWikiName($signupurl, WN_CAMEL_CASE_EVOLVED)) {	
```
devrait aussi être suffisant et être plus facile à lire

Reply:
Je préférais éviter les bugs dans le futur au cas ou "0" soit à un moment donné un wikiname valide. Ici, je suis explicite.

Reply:
en fait, si tout le bloc est dans un 
```
if ($signupurl !== "0"){
 ...
}
```
ça permet de retirer tous les tests multiples et ça aide à la lecture pour le debug (je pense à réduire la dette technique qui sera virer dès qu'il y aura refactor)

Reply:
c'est un détail pour moi. C'est comme tu veux.